test/filters/test_convolution.cpp

```diff
@@ -3155,6 +3134,2374 @@ main (int argc, char** argv)
+  (*input) (47,11).r = 6; (*input) (47,11).g = 181; (*input) (47,11).g = 207;
+  (*input) (47,12).r = 6; (*input) (47,12).g = 181; (*input) (47,12).g = 207;
+  (*input) (47,13).r = 6; (*input) (47,13).g = 181; (*input) (47,13).g = 207;
+  (*input) (47,14).r = 4; (*input) (47,14).g = 182; (*input) (47,14).g = 206;
+  (*input) (47,15).r = 4; (*input) (47,15).g = 182; (*input) (47,15).g = 206;
+  (*input) (47,16).r = 4; (*input) (47,16).g = 182; (*input) (47,16).g = 206;
+  (*input) (47,17).r = 2; (*input) (47,17).g = 183; (*input) (47,17).g = 204;
+  (*input) (47,18).r = 1; (*input) (47,18).g = 183; (*input) (47,18).g = 202;
+  (*input) (47,19).r = 1; (*input) (47,19).g = 183; (*input) (47,19).g = 202;
+  (*input) (47,20).r = 0; (*input) (47,20).g = 184; (*input) (47,20).g = 201;
+  (*input) (47,21).r = 0; (*input) (47,21).g = 184; (*input) (47,21).g = 201;
+  (*input) (47,22).r = 0; (*input) (47,22).g = 184; (*input) (47,22).g = 201;
+  (*input) (47,23).r = 0; (*input) (47,23).g = 185; (*input) (47,23).g = 199;
+  (*input) (47,24).r = 0; (*input) (47,24).g = 185; (*input) (47,24).g = 199;
+  (*input) (47,25).r = 0; (*input) (47,25).g = 184; (*input) (47,25).g = 201;
+  (*input) (47,26).r = 0; (*input) (47,26).g = 184; (*input) (47,26).g = 201;
+  (*input) (47,27).r = 0; (*input) (47,27).g = 184; (*input) (47,27).g = 201;
+  (*input) (47,28).r = 1; (*input) (47,28).g = 183; (*input) (47,28).g = 202;
+  (*input) (47,29).r = 1; (*input) (47,29).g = 183; (*input) (47,29).g = 202;
+  (*input) (47,30).r = 2; (*input) (47,30).g = 183; (*input) (47,30).g = 204;
+  (*input) (47,31).r = 4; (*input) (47,31).g = 182; (*input) (47,31).g = 206;
+  (*input) (47,32).r = 4; (*input) (47,32).g = 182; (*input) (47,32).g = 206;
+  (*input) (47,33).r = 4; (*input) (47,33).g = 182; (*input) (47,33).g = 206;
+  (*input) (47,34).r = 6; (*input) (47,34).g = 181; (*input) (47,34).g = 207;
+  (*input) (47,35).r = 6; (*input) (47,35).g = 181; (*input) (47,35).g = 207;
+  (*input) (47,36).r = 6; (*input) (47,36).g = 181; (*input) (47,36).g = 207;
+  (*input) (47,37).r = 6; (*input) (47,37).g = 181; (*input) (47,37).g = 207;
+  (*input) (47,38).r = 6; (*input) (47,38).g = 181; (*input) (47,38).g = 207;
+  (*input) (47,39).r = 6; (*input) (47,39).g = 181; (*input) (47,39).g = 207;
+  (*input) (47,40).r = 8; (*input) (47,40).g = 180; (*input) (47,40).g = 209;
+  (*input) (47,41).r = 8; (*input) (47,41).g = 180; (*input) (47,41).g = 209;
+  (*input) (47,42).r = 8; (*input) (47,42).g = 180; (*input) (47,42).g = 209;
+  (*input) (47,43).r = 8; (*input) (47,43).g = 180; (*input) (47,43).g = 209;
+  (*input) (47,44).r = 8; (*input) (47,44).g = 180; (*input) (47,44).g = 209;
+  (*input) (47,45).r = 8; (*input) (47,45).g = 180; (*input) (47,45).g = 209;
+  (*input) (47,46).r = 8; (*input) (47,46).g = 180; (*input) (47,46).g = 209;
+  (*input) (47,47).r = 8; (*input) (47,47).g = 180; (*input) (47,47).g = 209;
```

Comment:
These lines have a pattern. Moreover, the significance of the input is useful (aka they convolve to 0)

Reply:
Actually there is no special pattern. I produced the image data with MATLAB from random values. But I created a mathematical description and modified the test accordingly.